src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+const rows = [...tbody.children];
+let sortedRows = [];
+
+document.querySelector('thead').addEventListener('click', e => {
+  sortTable(e.target.textContent);
+});
+
+function sortTable(textContent) {
+  switch (textContent) {
+    case 'Name':
+      sortedRows = rows.sort((a, b) =>
+        a.children[0].textContent.localeCompare(b.children[0].textContent));
+      break;
+    case 'Position':
+      sortedRows = rows.sort((a, b) =>
+        a.children[1].textContent.localeCompare(b.children[1].textContent));
+      break;
+    case 'Age':
+      sortedRows = rows.sort((a, b) => +a.children[2].textContent
+        - +b.children[2].textContent);
+      break;
+    case 'Salary':
+      sortedRows = rows.sort((a, b) =>
+        +a.children[3].textContent.replace(/[^0-9.-]+/g, '')
```

Comment:
`/[^0-9.-]+/g` - move to variable

Reply:
Here I used a function instead of a variable